packages/filebrowser/src/crumbs.ts

```diff
@@ -370,9 +370,9 @@ namespace Private {
    */
   export function createCrumbSeparators(): ReadonlyArray<HTMLElement> {
     let items: HTMLElement[] = [];
-    for (let i = 0; i < 3; i++) {
-      let item = document.createElement('i');
-      item.className = 'fa fa-angle-right';
```

Comment:
Huh. I was under the impression we no longer were using font-awesome.

Reply:
Yeah, I was surprised too.